pkg/kubelet/dockertools/manager.go

```diff
@@ -369,11 +370,29 @@ func (dm *DockerManager) inspectContainer(dockerID, containerName, tPath string,
 	}
 
 	glog.V(4).Infof("Container inspect result: %+v", *inspectResult)
+
+	// Get restartCount from docker label, and add into the result.
+	// If there is no restart count label in an container:
+	//	1. It is an infraContainer, it will never use restart count.
+	//	2. It is an old container or an invalid container, we just set restart count to 0 now.
+	//	TODO (random-liu) Figure out how to deal with old containers with no restart count label.
```

Comment:
So far we've only promised "best-effort" restart count reporting, so I think it's okay to assume 0 when dealing with old containers.